src/repositories/friendship.repository.ts

```diff
@@ -17,6 +17,122 @@ export class FriendshipRepository extends Repository {
     return this.change('destroy', id, mediaIdAttribution);
   }
 
+  async removeFriendship(id: string | number, mediaIdAttribution?: string) {
+    return this.change('remove_follower', id, mediaIdAttribution);
+  }
+
+  async muteUserMedia(id: string | number, option: string, mediaIdAttribution?: string) {
```

Comment:
`option` type should be literal
```ts
option: 'story' | 'post'
```
So you don't need `switch` with `default`, because user can't pass other option, `if` will do the work. Compile-time check is always preferred way over run-time check.
Plus the `option` name is not self-described, user can't understand the meaning of this option in IDE. Should be something like `targetType` or anything you like.

Reply:
Method name should be `mutePostsOrStoryFromFollow `